datalad/tests/utils.py

```diff
@@ -469,7 +469,12 @@ def newfunc(*arg, **kw):
         flavors_ = _get_resolved_flavors(flavors)
 
         testrepos_uris = _get_testrepos_uris(regex, flavors_)
-        assert(testrepos_uris)
+        # MIH: I don't think we can assert that there always is a testrepo
+        # for network test when NONETWORK is given, this list will be empty
+        # will change it to skipping the test. Remove on ACK.
+        #assert(testrepos_uris)
```

Comment:
But is doesn't happen ;-)
On Sep 13, 2015 10:35 AM, "bpoldrack" <notifications@github.com> wrote:

> In datalad/tests/utils.py
> <https://github.com/datalad/datalad/pull/200#discussion_r39345714>:
>
> > @@ -469,7 +469,12 @@ def newfunc(*arg, **kw):
> >          flavors_ = _get_resolved_flavors(flavors)
> >
> >          testrepos_uris = _get_testrepos_uris(regex, flavors_)
> > -        assert(testrepos_uris)
> > +        # MIH: I don't think we can assert that there always is a testrepo
> > +        # for network test when NONETWORK is given, this list will be empty
> > +        # will change it to skipping the test. Remove on ACK.
> > +        #assert(testrepos_uris)
>
> Actually, for "with_testrepos" there's a parameter "flavor". Set to
> 'local_url' in case of DATALAD_NONETWORK. Then testrepos_uris should not be
> empty, but deliver local urls (file:///where/ever)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/datalad/datalad/pull/200/files#r39345714>.
>